inc/ThirdParty/Themes/Divi.php

```diff
@@ -118,6 +118,6 @@ public function add_divi_to_description( $disable_youtube_lazyload ) {
 	private static function is_divi( $theme = null ) {
 		$theme = $theme instanceof WP_Theme ? $theme : wp_get_theme();
 
-		return ( 'Divi' === $theme->get( 'Name' ) || 'divi' === $theme->get_template() );
+		return ( 'Divi' === $theme->get( 'Name' ) || 'Divi' === $theme->get_template() );
```

Comment:
```suggestion
		return ( 'divi' === strtolower( $theme->get( 'Name' ) ) || 'divi' === strtolower( $theme->get_template() ) );
```

With this change, we force the returned values to be lowercase, eliminating the naming issue of the parent/child theme. This improves consistency on our side of the code.